client/driver/rkt.go

```diff
@@ -240,7 +243,20 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 	img := driverConfig.ImageName
 
 	// Build the command.
-	cmdArgs := make([]string, 0, 32)
+	cmdArgs := make([]string, 0, 50)
+
+	// Run rkt with env command to inject PATH as rkt needs to be able to find iptables
+	envAbsPath, err := GetAbsolutePath(envCmd)
```

Comment:
Switched it to use a fresh env with just the host env vars on it. Seems more reasonable than mutating TaskEnv to add the PATH in just for the rkt command.